src/modules/activity.js

```diff
@@ -0,0 +1,55 @@
+import { getItemFromLocalStorage, saveItemToLocalStorage } from './storage.js';
+
+const createTask = () => {
+  const tasks = getItemFromLocalStorage();
+  const newTaskDescription = document.getElementById('to-do-input');
+  const newTask = {
+    description: newTaskDescription.value,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  newTaskDescription.value = '';
+  tasks.push(newTask);
+
+  saveItemToLocalStorage(tasks);
+};
+
+const updateTask = (taskId, el) => {
+  const tasks = getItemFromLocalStorage();
+  const task = tasks.find((task) => task.id === parseInt(taskId, 10));
+  if (el.hasAttribute('content-editatable')) {
+    task.description = el.textContent;
+  } else {
+    const span = el.nextElementSimbling;
+    const parent = el.closest('input');
+    task.isCompleted = !task.isCompleted;
+    if (task.isCompleted) {
+      span.removeAttribute('content-editable');
+      span.classList.add('complete');
+    } else {
+      span.setAttribute('content-editable', 'true');
+      parent.classList.remove('complete');
+    }
+  }
+  localStorage.setItem('task', JSON.stringify(this.tasks));
+};
+
+const deleteTask = (tasks, index) => {
+  const newTasks = tasks.filter((task, innerIndex) => index !== innerIndex);
+  for (let i = 0; i < newTasks.length; i += 1) {
+    newTasks[i].index = tasks.length;
+  }
```

Comment:
I would advise you to review this for loop statement and make an attempt to replace with a forEach loop